### ISSUE TITLE
Config not working (v1.8.0)

### ISSUE THREAD

**Issue by rtejano1207** [kapitainsky/RcloneBrowser#104]
I am not sure if this is a bug but I am getting a blank response when I clicked CONFIG.  A window appears and closes momentarily.

the following applications are installed.
rclone browser: v1.8.0
rclone: v1.51
os/arch: linux/amd64 (linux mint 19.3)
go: 1.13.7

I can do a manual rclone config via terminal but the said configuration does not appear in UI configuration list.

![Screenshot from 2020-04-11 11-24-53](https://user-images.githubusercontent.com/15810256/79034332-b9051000-7be7-11ea-831b-185847dde1a1.png)
![Screenshot from 2020-04-11 11-25-39](https://user-images.githubusercontent.com/15810256/79034339-c02c1e00-7be7-11ea-84f2-ffba736d72c2.png)

**Comment by kapitainsky** [comment]
"Config" button issue would indicate that there is problem starting terminal (at least from Rclone Browser perspective). And by any chance do you have $TERMINAL environment variable configured in your Linux? I am thinking if it maybe points to some exotic terminal..

Besides that issue it should not affect Rclone Browser if you preformed rclone configuration manually already. Could you check if rclone lists remotes when you run from command line `rclone listremotes`? Doesn't pressing "Refresh" button in Rclone Browser help? would you mind to post here your rclone config? (remove any password/token info)

**Comment by rtejano1207** [comment]
Existing TERM variable is pointing to xterm 
I added new environment variable TERMINAL=xterm
Clicking "Config" from Rclone Browser 1.8 pops up a terminal window and it momentarily closes the same.  The terminal shows permission denied reading rclone.conf

Running rclone listremotes from terminal returns a list of the manually created configuration.

Pressing "Refresh" from RClone Browser does not list the manually created configuration.

Could this be permission related?

![Screenshot from 2020-04-12 11-02-52](https://user-images.githubusercontent.com/15810256/79059689-91797a80-7caf-11ea-8040-75e4b88c0b74.png)
![Screenshot from 2020-04-12 11-08-03](https://user-images.githubusercontent.com/15810256/79059693-9a6a4c00-7caf-11ea-8deb-ff7a7c5a8270.png)

**Comment by kapitainsky** [comment]
I have tried on my Linux to replicate the issue. I've got the same Rclone Browser behaviour (shows rclone version, does not list remotes and fails to start config) exactly in situation when defined rclone.conf does exist but I have no permission to read it. But then also running `rclone listremotes` shows error message "rclone.conf: permission denied" - when in your case it works.

Now it is possible when you run different rcone from command line than from RB... I see that you configured /usr/bin/rclone in your RB preferences.

Let's try this one explicitly from cmd `/usr/bin/rclone listremotes -vv`
-vv option will display debug info including rclone.conf used e.g.:

```
db@DB-laptop-01:~/.config/rclone$ rclone listremotes -vv
2020/04/12 07:45:11 DEBUG : rclone: Version "v1.51.0" starting with parameters ["rclone" "listremotes" "-vv"]
2020/04/12 07:45:11 DEBUG : Using config file from "/Users/db/.config/rclone/rclone.conf"
```

is it the same as configured in preferences?

You can also try to remove `rclone.conf location` from RB preferences (default rclone location then will be used).

**Comment by kapitainsky** [comment]
Also FYI my /home/db/.config/rclone.conf permissions:

`[db@localhost rclone]$ ll
total 8
-rw-------. 1 db db 4137 Apr 10 11:00 rclone.conf
`

**Comment by rtejano1207** [comment]
Issuing chmod a+r rclone.conf did the trick for me.  RB is now able to read and launch Config flawlessly.

I am not sure  if there is any security implication by the above command but i only added read rights for all.  Can you confirm from your end as well?  I hope there could be a better workaround directly from RB's end on this matter.   Nonetheless you have a great product.

![Screenshot from 2020-04-13 07-52-50](https://user-images.githubusercontent.com/15810256/79082856-a6f8ae00-7d5c-11ea-8090-d80591e71285.png)
![Screenshot from 2020-04-13 07-53-12](https://user-images.githubusercontent.com/15810256/79082860-afe97f80-7d5c-11ea-805e-16560d3ff7de.png)

**Comment by kapitainsky** [comment]
This is definitely not standard behaviour I have never seen situation like this. Of course rclone needs access to its config file to work. Under normal circumstances user has read/write access to this file, then the same user runs RB so RB has inherited rights (passed to invoked rclone). 

In your case rclone.conf file is owned by user raymond and without read permissions for all RB does not have rights to read it. What does it mean? I am thinking that your RB is run as another user. Could you check it?

Run `ps -ef | grep rclone`

When on Linux I run RB as user db it shows:

`db        11231      1  0 01:13 ?        00:00:00 ./rclone-browser-1.9.8.0-69f1d5c-linux-x86_64.AppImage
`
so my RB instance has no problems with accessing file owned by user db.

**Comment by rtejano1207** [comment]
RB is owned by the user raymond.  I forgot to inform earlier that rclone.conf was previous owned by root and not by raymond so I had to issue chmod +r

But taking cue from your end that both your RB and rclone.conf are owned by db user but still it does not list  the remotes so I had to do the above workaround from my end.

![Screenshot from 2020-04-13 08-27-03](https://user-images.githubusercontent.com/15810256/79083507-47e96800-7d61-11ea-94bc-276ec4f661ed.png)

**Comment by kapitainsky** [comment]
question is not who own rclone browser file but who runs it...

Start RB and the in terminal check `ps -ef | grep rclone`

**Comment by rtejano1207** [comment]
RB is runnig with user raymond

![Screenshot from 2020-04-13 08-49-02](https://user-images.githubusercontent.com/15810256/79083956-d65ee900-7d63-11ea-9d87-deb4a0107ff0.png)

**Comment by kapitainsky** [comment]
ok - but I wonder why there are two instances? On my system I can see only one.

**Comment by rtejano1207** [comment]
Yes.  this is also an observation.  I only launched the RB once but showed two processes. If i close the app, both process also close.

**Comment by kapitainsky** [comment]
maybe you are using some AppImage management software.

**Comment by rtejano1207** [comment]
I haven't installed yet AppImageLauncher.  I launched RB direclty.

**Comment by sdotb** [comment]
output of `which rclone`?
check if `/usr/local/bin/rclone` or `/urs/bin/rclone`

**Comment by rebelclause** [comment]
The same problem over here. RB stopped prompting for a password and shows no Remotes, though Tasks are still there and editable. rclone config has always required sudo to run and the config remains accessible with one of the scripts created for the CLI syncs properly to the remote it has a token to access. On trying to launch Config from RB's main screen, what appears to be an XTERM pops open, then immediately closes. No joy on resolving this using the latest AppImage 1.80, or upgrading rclone to the latest (with rclone continuing to work as described above). In the RB ~/.config/rclone-browser directory there are historical config files and one with .save on its end. Quitting RB, the incumbent .conf was tagged .old and the most recent legacy file put into current position. The same problem. I've recently also had docker-compose suddenly stop working with a hacky kind of error on timeout values showing to be int, not being so. And because these two events occurred one after another between reboots (docker first and then rclone-browser), I get a sand vine vibe, like their two particles in the same pool of quicksand. I ran ps -ef | grep rclone, and lo, for the earlier version 1.7.0 and for the latest as of this writing, 1.8.x, rclone-browser had two instances to report. Launching rclone config and doing ps -ef again, rclone showed up as expected with the same result on rclone-browser. Similarly, listremotes show all the expected remotes.

**Comment by kapitainsky** [comment]
I am trying to replicate it.

**Comment by Thanatoslayer6** [comment]
The same thing happened to me recently however, I managed to fix it by installing xterm and unsetting my $TERM environment variable, and with that I rebooted my pc and it worked. I use the appimage version though, I hope this helps :)
